modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/display/context/DLOpenerOneDriveDLViewFileVersionDisplayContext.java

```diff
@@ -165,6 +205,30 @@ public List<ToolbarItem> getToolbarItems() throws PortalException {
 		return toolbarItems;
 	}
 
+	private List<DropdownItem> _addEditInOffice365DropdownItem(
+		List<DropdownItem> dropdownItems,
+		DropdownItem editInOffice365DropdownItem) {
+
+		int i = 1;
+
+		for (DropdownItem dropdownItem : dropdownItems) {
```

Comment:
If the dropdownItem is a group the option is added at the end of the dropdown instead down the edit option as before

![image](https://user-images.githubusercontent.com/47524751/176461456-dec16a8a-f2dd-486b-92fa-4214507c2d56.png)

but with this PR it shows at the final

![image](https://user-images.githubusercontent.com/47524751/176461805-53aad0e0-17fb-44ee-a2d8-b12510741744.png)

@adolfopa I'll send 2 commits to make this, please review it